### PR TITLE
Closing # 87 - updated formatting, contents and images

### PR DESCRIPTION
**Pull request by Kalyan-Lama**

### CODE REVIEW

**Comment by Kalyan-Lama** [comment]
@joshuaberetta, I have updated this support article. Could you kindly review and merge them with the master branch? cc: @tinok

File: source/pulling_data_into_excelquery.md [hunk 1]
```diff
@@ -1,44 +1,73 @@
...
+
+* You will now be able to see a dialogue box, `Data source settings`. Here, you will need to press `Clear Permissions`. With this, you should be able to see the dialogue box to input your login credentials.
+
+![image](/images/pulling_data_excelquery/data_source_settings.png)
+
+If you wish to learn more on resetting your Excel's `Data Source Settings` feel free to explore [here](https://docs.microsoft.com/en-us/power-query/connectorauthentication).
+
+### Updating Data in Microsoft Excel
+
+Once you have synced your KoBoToolbox with your Microsoft Excel through __Excel Power Query__, you should be able to have the dataset in Microsoft Excel. However, when the KoBoToolbox server is updated (i.e. through new submissions, deletion, amendments) your Microsoft Excel dataset does not get updated automatically. You could do this by refreshing the Excel document. For this you will need to click the **Data** and then select **Refresh All**. Any changes made in the KoBoToolbox server will now be manually updated in your Excel dataset. (Please follow this whenever you require an update with your Microsoft Excel dataset)
```
Comment (joshuaberetta) [inline source/pulling_data_into_excelquery.md:71]: You can rephrase this to something like:

> Once you have synced KoBoToolbox with Excel through __Excel Power Query__, you should have the dataset stored locally. However, when the dataset on the server updates (i.e. through new submissions, deletions, amendments), your local dataset does not automatically update. To resync your data, you will need to click the __Data__ button and then __Refresh All__.

**Review by joshuaberetta — CHANGES_REQUESTED** [review]

**Review by joshuaberetta — APPROVED** [review]
